packages/service-utils/src/core/services.ts

```diff
@@ -51,6 +51,13 @@ export const SERVICE_DEFINITIONS = {
     // all actions allowed
     actions: [],
   },
+  chainsaw: {
+    name: "chainsaw",
+    title: "Chainsaw",
```

Comment:
probably not the public name?

Reply:
Probably not, but can be changed after. Don't want to wait on naming.

Figured out api-server needs changes too so just created a PR in api-server

Reply:
yep, doesn't show in UI for now, right?

Reply:
and api server PR makes sense, however I think you'll have to add it to `SERVICES` in service utils (in this repo), too?

Reply:
`SERVICES` are generated from `SERVICE_DEFINITIONS`, if it's the same file you mean

https://github.com/thirdweb-dev/js/blob/main/packages/service-utils/src/core/services.ts#L60

Reply:
Added chainsaw to HIDDEN_SERVICES [857b040](https://github.com/thirdweb-dev/js/pull/3745/commits/857b040b76e56589b5ac8425f83b500c82eb361d)